test-content/README.md

```diff
@@ -70,11 +77,11 @@ the git filter-branch command...?
 
 <h3 id="lpmi-13"><a href="https://github.com/lpmi-13">Adam</a></h3>
 
-- [questions](https://github.com/asdf/asfd/issues/?q=author%3Alpmi-13+label%3Aquestion)
+- [questions](https://github.com///issues/?q=author%3Alpmi-13+label%3Aquestion)
   |
-  [help-wanted](https://github.com/asdf/asfd/issues/?q=author%3Alpmi-13+label%3Ahelp-wanted)
+  [help-wanted](https://github.com///issues/?q=author%3Alpmi-13+label%3Ahelp-wanted)
```

Comment:
if we need this `///`, could we possibly add a comment for the reason why it's needed? I'm thinking the distinction between places that need `undefined` and places that need `///` might not be obvious.